dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -786,21 +786,21 @@ BEGIN
     IDS_ABOUT_VERSION_STRING "Versiune %S (%S)"
 
     /* context menus */
-    IDS_VIEW_LARGE "D&ale"
+    IDS_VIEW_LARGE "&Dale"
     IDS_VIEW_SMALL "&Pictograme"
     IDS_VIEW_LIST "&Listă"
     IDS_VIEW_DETAILS "&Detalii"
     IDS_SELECT "Selectea&ză"
     IDS_OPEN "&Deschide"
     IDS_CREATELINK "Creează sc&urtătură"
-    IDS_COPY "&Copiază"
-    IDS_DELETE "Șt&erge"
+    IDS_COPY "&Copiere"
+    IDS_DELETE "Ş&tergere"
     IDS_PROPERTIES "&Proprietăți"
```

Comment:
You chose
MENUITEM "Prop&rietăţi", IDM_PROPERTIES
above
so you must here also use
IDS_PROPERTIES "P&roprietăți"

Table at the top and stuff used here in this section must always match!

Reply:
Resolved.